index.html

```diff
@@ -4,27 +4,40 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
-    <title>Welcome to my mobile portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <title>Welcome to my portfolio website</title>
 </head>
 <body>
-<div class="container">
+
+    <!-- Tollbar and Header section starts from here -->
+
+    <div class="container">
     <header class="toolbar-container">
-        <a class="welcome_logo">
-            <img src="images/Welcome logo.svg">
+        <a href="index.html">
+            <img id="logo-image" src="images/logo.svg" alt="Logo Image" />
         </a>
         <nav class="menu">
-            <img src="images/Menu.svg" alt="icon_menu">
+            <img src="images/menu.svg" alt="Menu_icon">
+        </nav>
+        <nav class="navbar">
+            <ul id="navbar-Pages">
+            <li><a href="#" class="menu-item-1">Portfolio</a></li>
+            <li><a href="#" class="menu-item-2">About</a></li>
+            <li><a href="#" class="menu-item-3">Contact</a></li>
+            </ul>
         </nav>
         </header>
         <section class="hero-title-section">
-        <h1 class="hero-title">
-    Hey there. I'm Jerome<br>
-    <span>I'm a software developer</span>
-        </h1>
-        <p class="hero-description">
-            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don't hestiate to contact me.</p>
+        <img id="headline-img-mobile" src="images/header-illsutration-mobile 1.svg" alt="Mobile header background image">
+        <img id="headline-img-desktop" src="images/Header-llustration-desktop.svg" alt="Desktop header background image">
```

Comment:
- [ ] Kindly note that all images meant for decoration purposes should be added as `background-image` in your CSS file. Kindly move these images to your CSS file. This is best practice. Kindly check [here](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more details.